extensions-core/datasketches/src/test/java/org/apache/druid/query/aggregation/datasketches/tuple/ArrayOfDoublesSketchAggregationTest.java

```diff
@@ -200,6 +200,7 @@ public void ingestingSketchesTwoValues() throws Exception
             "      \"name\": \"union\",",
             "      \"operation\": \"UNION\",",
             "      \"nominalEntries\": 1024,",
+            "      \"numberOfValues\": 2,",
```

Comment:
can you please explain this change? given that everything is backwards compatible why did this have to change?

Reply:
This was a mistake in the test. The number of values in the union and incoming sketches must match, which was not enforced before. Correct code should not be affected.

Reply:
ok